src/test/java/org/opentripplanner/gtfs/mapping/TripMapperTest.java

```diff
@@ -111,22 +111,22 @@ void testMapCache() throws Exception {
   }
 
   @Test
-  void noFlexDurationModifier() {
+  void noFlexTimePenalty() {
     var mapper = defaultTripMapper();
     mapper.map(TRIP);
-    assertTrue(mapper.flexSafeDurationModifiers().isEmpty());
+    assertTrue(mapper.flexSafeTimePenalties().isEmpty());
   }
 
   @Test
-  void flexDurationModifier() {
+  void flexTimePenalty() {
     var flexTrip = new Trip();
     flexTrip.setId(new AgencyAndId("1", "1"));
     flexTrip.setSafeDurationFactor(1.5);
     flexTrip.setSafeDurationOffset(600d);
     flexTrip.setRoute(new GtfsTestData().route);
     var mapper = defaultTripMapper();
     var mapped = mapper.map(flexTrip);
-    var mod = mapper.flexSafeDurationModifiers().get(mapped);
+    var mod = mapper.flexSafeTimePenalties().get(mapped);
```

Comment:
this variable could be renamed

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/5856/commits/57c4282909bc05ef1eee427954639d780b2e2659